pyodbc.cpython-313-x86_64-linux-gnu.so,sha256=0td1lyjsq1sxybfGRYJU0xUpWQAvkmlee6yMXhOA1lo,987384
pyodbc.pyi,sha256=YSHcS8DHzgggA5eKVJuBpvz6RAOSaO39BqTUEl7GA2c,32673
pyodbc-5.2.0.dist-info/top_level.txt,sha256=Rb69vS-SNN_Bfh99vX7eCcvEAJoyzowG3u4R7W36Flw,8
pyodbc-5.2.0.dist-info/RECORD,,
pyodbc-5.2.0.dist-info/LICENSE.txt,sha256=mFnIV3DJG3hmsPdpyTjjbWTQJgFBtSm5R7Tq2WiD6rQ,868
pyodbc-5.2.0.dist-info/METADATA,sha256=CSPn1FnJoZwQCr5qahAMjloJIeVYFsdjtSAMxpLZHVU,2742
pyodbc-5.2.0.dist-info/WHEEL,sha256=l8JebZ2EucL4EHBzvkCTCBq8pKOzA1XejIK_DsvkSYE,151
